doc/source/methods/ProtoSelect.ipynb

```diff
@@ -123,36 +123,42 @@
     "\n",
-    "Finally, we can obtain the explanation by requesting the maximum number of prototypes to be returned:\n",
+    "Finally, we can obtain summary by requesting the maximum number of prototypes to be returned:\n",
     "\n",
     "```python\n",
     "summary = summariser.summarise(num_prototypes=num_prototypes)\n",
     "```\n",
     "\n",
-    "* `num_prototypes`: Number of maximum prototypes to be selected.\n",
+    "* `num_prototypes`: Maximum number of prototypes to be selected.\n",
     "\n",
     "As we previously mentioned, the algorithm stops when the objective is less than 0, for all the remaining instances in the set of potential prototypes. This means that the algorithm can return a lower number of prototypes than the one requested.\n",
     "\n",
-    "Another important observation is that the summary returns the prototypes with their corresponding labels although no labels were provided for the set `X`. This is possible since each prototype $x$ will belong to a prototype set $\\mathcal{P}_l$, and thus we can assign a label $l$ to $x$. Following the summarisation step, one can train an interpretable 1-KNN classifier on the returned prototypes even for an unlabeled dataset `X`. "
+    "Another important observation is that the summary returns the prototypes with their corresponding labels although no labels were provided for $Z$. This is possible since each prototype $z$ will belong to a prototype set $\\mathcal{P}_l$, and thus we can assign a label $l$ to $z$. Following the summarisation step, one can train an interpretable 1-KNN classifier on the returned prototypes even for an unlabeled dataset $Z$.\n",
+    "\n",
+    "<div class=\"alert alert-block alert-warning\">\n",
+    "<b>Warning</b> \n",
+    "    \n",
+    "If the optional argument $Z$ is not provided, it is automatically set to $X$. Although the labels of the data instances belonging to $Z$ are available in this case, the dataset $Z$ is still viewed as an unlabeled dataset. This means that a prototype $z_i \\in Z$ belonging to the class $l$ according to the labels $y$, can be a prototype for a class $k \\ne l$. \n",
+    "\n",
+    "</div>"
```

Comment:
This is not rendering properly, have a look at the source code of an existing notebook doc that uses these directives, e.g. https://docs.seldon.io/projects/alibi/en/latest/methods/TreeSHAP.html